ClientApp/Podfile

```diff
@@ -0,0 +1,46 @@
+source 'https://github.com/firebase/SpecsDev.git'
```

Comment:
First two files shouldn't be added. Maybe the result of a bad merge after the Integration Tests were moved to the IntegrationTesting directory?